features/application-mgt/org.wso2.carbon.identity.application.mgt.server.feature/resources/identity/authenticationTemplates/roleV2.json

```diff
@@ -0,0 +1,55 @@
+{
+  "category": "AdaptiveMFA",
+  "name": "Role-V2-Based",
```

Comment:
should we hide role v2 templates with the legacy runtime?

Reply:
updated